docs/tutorial/index.md

```diff
@@ -0,0 +1,527 @@
+
+![Create package profile](./assets/create-package-profile.png)
+
+At the bottom of the "Vision" profile page, in the "Association" section, fill in the "New tags" field with the value
+`wsl-vision` we used to tag the instances when they were accepted in Landscape and click "Change".
+
+![Applying the profile to the WSL instances](./assets/applying-profile.png)
+
+In the "Summary" section in the middle of the page a status message will show that 2 computers are `applying the profile`. Click on the `applying the profile` link and then, in the "Activities" list, click on "Apply package profile" to see the progress of the package deployment.
+
+![Progress of the package deployment](./assets/package-deployment-progress.png)
+
+Eventually the instances will report having succeeded in applying the profile. Verify in the instances shells that the `python3-opencv` package was installed.
+For example, in the `Ubuntu` instance it would look like:
+
+```bash
+me@mib:~$ apt list --installed | grep opencv
+
+WARNING: apt does not have a stable CLI interface. Use with caution in scripts.
+
+libopencv-calib3d4.5d/jammy,now 4.5.4+dfsg-9ubuntu4 amd64 [installed,automatic]
+libopencv-contrib4.5d/jammy,now 4.5.4+dfsg-9ubuntu4 amd64 [installed,automatic]
+libopencv-core4.5d/jammy,now 4.5.4+dfsg-9ubuntu4 amd64 [installed,automatic]
+libopencv-dnn4.5d/jammy,now 4.5.4+dfsg-9ubuntu4 amd64 [installed,automatic]
+libopencv-features2d4.5d/jammy,now 4.5.4+dfsg-9ubuntu4 amd64 [installed,automatic]
+libopencv-flann4.5d/jammy,now 4.5.4+dfsg-9ubuntu4 amd64 [installed,automatic]
+libopencv-highgui4.5d/jammy,now 4.5.4+dfsg-9ubuntu4 amd64 [installed,automatic]
+libopencv-imgcodecs4.5d/jammy,now 4.5.4+dfsg-9ubuntu4 amd64 [installed,automatic]
+libopencv-imgproc4.5d/jammy,now 4.5.4+dfsg-9ubuntu4 amd64 [installed,automatic]
+libopencv-ml4.5d/jammy,now 4.5.4+dfsg-9ubuntu4 amd64 [installed,automatic]
+libopencv-objdetect4.5d/jammy,now 4.5.4+dfsg-9ubuntu4 amd64 [installed,automatic]
+libopencv-photo4.5d/jammy,now 4.5.4+dfsg-9ubuntu4 amd64 [installed,automatic]
+libopencv-shape4.5d/jammy,now 4.5.4+dfsg-9ubuntu4 amd64 [installed,automatic]
+libopencv-stitching4.5d/jammy,now 4.5.4+dfsg-9ubuntu4 amd64 [installed,automatic]
+libopencv-video4.5d/jammy,now 4.5.4+dfsg-9ubuntu4 amd64 [installed,automatic]
+libopencv-videoio4.5d/jammy,now 4.5.4+dfsg-9ubuntu4 amd64 [installed,automatic]
+libopencv-viz4.5d/jammy,now 4.5.4+dfsg-9ubuntu4 amd64 [installed,automatic]
+python3-opencv/jammy,now 4.5.4+dfsg-9ubuntu4 amd64 [installed,automatic]
+```
+
+Good job! Your instances are transparently managed by Landscape thanks to the automation provided by Ubuntu Pro for WSL.
+
+## Conclusion
+
+In this tutorial you learned how to ensure new Ubuntu WSL instances are automatically attached to Ubuntu Pro and
+registered into a self hosted Landscape server thanks to the automation provided by Ubuntu Pro for WSL.
+Here is a complete overview of what we just achieved:
+
+[Diagram]
```

Comment:
Add a <!-- TODO: bla bla bla --> so that we can grep for this sorts of things latter.

Reply:
Funny, if don't use `tildes` GitHub hides the comment 😅 

<!-- I'm hidden 😈 -->